### PR TITLE
Update schema according to the changes in DPP official docs

### PR DESCRIPTION
**Pull request by krzysiekfonal**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -165,7 +158,7 @@ components:
           description: ISO timestamp
           example: 2019-10-12T07:20:50.52Z
           type: string
-        url:
+        paymentUrl:
```
Comment (jadwahab) [inline openapi.yaml:161]: i would leave this as just url so it's short and concise since it's clear that it's the payment url

File: openapi.yaml [hunk 2]
```diff
@@ -27,20 +27,13 @@ paths:
...
       responses:
-        "201":
+        "200":
           description: contains outputs, beneficiary data and expiry information, used by the
             payee to construct a transaction
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/dpp.PaymentTerms"
-        "400":
```
Comment (jadwahab) [inline openapi.yaml:37]: any reason why we would remove the 400 response?

Comment (krzysiekfonal) [reply openapi.yaml:37]: I think I reported this already somewhere in our conversations but this 400 is clearly a mistake - it is just copy&paste of 400 from the other endpoint - POST Payment. See the explanation of error:
"returned if the sender input is invalid, usually an issue with the paymentID" - it is not relevant for getting PaymentTerms. Getting provides only one url param - paymentID, so I think only 404 is relevant here.

File: openapi.yaml (hunk 1 above)
Comment (krzysiekfonal) [reply openapi.yaml:161]: Hmm but this is very small change/difference but having significant impact on backward compatibility. We've implemented DPP already and it works for many clients (wallets) which doesn't implemented it yet - they just understand the new PaymentTerms which contains output (the deprecated field, which was left for baxkward compatibility). So with this small change we will break this effort. And even I don't think it is really obvious it is payment url. As this is very extensible there might be more urls in the future (e.g. electrum guys already proposed refundUrl, actually in Originator not PaymentTerms but this is just example).

Comment (jadwahab) [reply openapi.yaml:161]: okay

**Review by jadwahab — APPROVED** [review]

**Pull request merged**